src/renderer/components/PerCurrencySelectAccount/state.js

```diff
@@ -46,6 +46,14 @@ const getIdsFromTuple = (accountTuple: AccountTuple) => ({
   subAccountId: accountTuple.subAccount ? accountTuple.subAccount.id : null,
 });
 
+export type useCurrencyAccountSelectReturnType = {
+  availableAccounts: Array<AccountTuple>,
+  currency: ?CryptoCurrency | TokenCurrency,
+  account: ?Account | any,
+  subAccount: ?SubAccount | any,
+  setAccount: (account: ?Account, subAccount: ?SubAccount) => void,
+  setCurrency: (currency: ?CryptoOrTokenCurrency) => void,
```

Comment:
Be explicit here and do `(CryptoCurrency | TokenCurrency)` as we do everywhere else too

Reply:
Done!